<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CriticalMsgBox</class>
 <widget class="QMainWindow" name="CriticalMsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #442726;
}

QLineEdit{
	background-color: #2F2F2F;
	border-radius: 3px;
	color: white;
	border-bottom: 1px;
	border-style: solid;
	border-color: #979797;
}

QLineEdit:focus{
	border-color: #60CDFF;
}

QPushButton
{
    color: black;
    font-size: 15px;
    background-color: #60CDFF;
    border-radius: 5px;
    font-family: &quot;Noto Sans&quot;
}

QPushButton::hover{ 
    background-color: #60CDFF;
}

QPushButton:pressed{
    border-width: 1px;
    border-style: solid;
	border-color: white;
}

QLabel
{
    color: white;
	font-size: 16px;
    font-family: &quot;Noto Sans&quot;;l
    border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>35</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Warning</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Are you sure you want to delete this book?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
